README.md

```diff
@@ -71,12 +28,86 @@ The web server will use the address value specified in the configuration file.
 - Visit <https://localhost:8000/graphql/playground> to access the GraphQL
   playground.
 
-## FLAGS
+### FLAGS
```

Comment:
To maintain consistency, `Flags` would look better.

Reply:
I've reflected the feedback.